ethers-contract/tests/it/abigen.rs

```diff
@@ -750,3 +750,21 @@ fn can_handle_overloaded_function_with_array() {
     ]"#,
     );
 }
+
+#[tokio::test]
+async fn convert_uses_correct_abi() {
```

Comment:
doesn't need to be async
```suggestion
#[test]
fn convert_uses_correct_abi() {
```